.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,40 @@
-<!-- Please replace #XX below with an existing issue number. Remove the line entirely if none exist. -->
-Fixes #XX
+## Fixes
+<!-- If PR only partly solves an issue, replace 'Fixes' below with 'Partially addresses'. -->
+<!-- If there is no issue being resolved, open one. -->
+Fixes #[issue number] by @[issue author]
```

Comment:
The issue author is already available in the issue itself. I don't think we need to replicate it here

Reply:
I think this is a good idea, it notifies the issue author that a PR addressing their issue has been opened. It's not just for the information.

Reply:
@kgodey that was the idea: https://github.com/creativecommons/creativecommons.github.io-source/pull/209/files#r398646000

Reply:
Yup, sounds good. Seems like @brenoferreira is fine with it since he approved the PR after my comment.